### ISSUE TITLE
[Bug] Comprehensive Operator Exception during Model vLLM Inference

### ISSUE THREAD

**Issue by shh2000** [flagos-ai/FlagGems#986]
### Describe the bug

Hardware: H20-SXM*8
Env: nvcr.io/nvidia/pytorch:25.06-py3
Framework: vllm in flagscale, varies with models
Preliminaries: every step mentioned below is correct without flag_gems.enable(). otherwise, we should first solve vllm's problem
1. How to Reproduce: vllm serve <model>, than test CMMMU and GPQA with opencompass&evalscope. If model cannot be served in 1 GPU, use vllm serve <model> --tensor-parallel-size 8
2. What is "exception": vllm serve process failed with OS exception during CMMMU and GPQA
3. How to determine the operator trigged the exception: use flag_gems.enable(record=True, once=False), than consider the last operator in recordFile should be unused. 
4. How to get exception operator list: Repeatedly execute step-3 until CMMMU (for LMM and omni) or GPQA(for LLM and omni) can be tested correctly

One Possible and easier reproduce way:
1. find Model on modelscope
2. execute command in Model's README, by which you can launch a correct server with flag_gems.enable(unused=[\<exception operators\>])
3. use pip show vllm, than cd vllm's root dir, use grep -rn "gems" ./ to find where flag_gems.enable() is
4. eliminate operator in unused, than you can launch the incorrect server, or directly trigger exception. Most of exceptions are triggered during CMMMU/GPQA, while others are triggered during launching process
5. test CMMMU/GPQA to trigger the exception

**Comment by shh2000** [comment]
Model: Kimi-K2-Thinking
Exact vllm version: 0.11.2
Exact serve command: vllm serve /share/Kimi-K2-Thinking   --served-model-name Kimi-K2-Thinking-nvidia-origin   --trust-remote-code   --tensor-parallel-size 8   --enable-auto-tool-choice   --max-num-batched-tokens 32768   --tool-call-parser kimi_k2   --reasoning-parser kimi_k2
Model Class: LLM
Datasets: GPQA
exception operator list: "exponential_", "fill_scalar_", "mul", "reciprocal", "true_divide", "sub", "ones", "mm", "mm_out", "sum_dim_out", "sum_dim_comm", "addmm_out", "full", "masked_fill_", "softmax", "argmax", "lt", "lt_scalar", "fill_tensor_", "to_dtype", "where_scalar_self", "where_self_out", "where_self", "true_divide_", "radix_sort", "sort", "sort_stable", "cumsum", "scan_then_fan", "cumsum_wrapper", "gather", "cumsum_out", "scatter"
